pkg/kubelet/cm/helpers_linux_test.go

```diff
@@ -49,21 +53,30 @@ func getResourceRequirements(requests, limits v1.ResourceList) v1.ResourceRequir
 }
 
 func TestResourceConfigForPod(t *testing.T) {
+	defaultQuotaPeriod := uint64(100 * time.Millisecond / time.Microsecond)
+	tunedQuotaPeriod := uint64(5 * time.Millisecond / time.Microsecond)
+	tunedQuota := int64(1 * time.Millisecond / time.Microsecond)
+
+	utilfeaturetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, pkgfeatures.CPUCFSQuotaPeriod, true)
```

Comment:
doesn't this stop us from testing the function when the feature gate is off?

shouldn't you have a seperate Test function that runs the test with it off, and another with it on?

Reply:
I duplicated the test and have one enabled and one disabled.
One problem for the test is that the ResourceConfigForPod() gets the period as paramter so the test will have expected the tunedQuotaPeriod because of that. The period parameter will not be passed into it, but this is outside of the cm package. 
I am not sure if this is nice to read in the test, maybe you have a better suggestion? 
I could add the feature gate in cm, such that the parameter will be ignored, which I think makes even less sense.